hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -117,6 +117,47 @@ vixl::Condition C(ConditionCode cc) {
   return arm::convertCC(cc);
 }
 
+/*
+ * Uses the flags from the Vinstr which defs SF to determine
+ * whether or not the Vixl assembler should emit code which
+ * sets the status flags.
+ */
+vixl::FlagsUpdate updateFlags(uint8_t flags) {
+  return flags ? SetFlags : LeaveFlags;
+}
+
+#define UF(x) updateFlags(x)
```

Comment:
These short macros (`UF` and `CSF`) are used for the one-line emitters to keep them under 80 columns wide.